Logger/BaseLoggerMixins.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Logger;
+
+public static class BaseLoggerMixins
+{
+    public static void Error(this BaseLogger baseLogger, string message, params object[] args)
```

Comment:
```suggestion
    public static void Error(this BaseLogger logger, string message, params object[] args)
```
Rename your variable to `logger` instead of `baseLogger` because this variable won't always, directly, be a BaseLogger... It could be a FileLogger which is a type of BaseLogger